test/e2e/main/main_test.go

```diff
@@ -817,3 +818,73 @@ func throwHTTPError(resp *http.Response) {
 	errorBytes, _ := ioutil.ReadAll(resp.Body)
 	panic(fmt.Errorf("unexpected HTTP status code (%d): %s", resp.StatusCode, string(errorBytes)))
 }
+
+func TestCacheCR(t *testing.T) {
+	spec := tutils.DefaultSpec(testKube)
```

Comment:
@durgeshanaokar can we run both test in parallel?

Reply:
@dmvolod can you please review this change.

Reply:
That's good, @durgeshanaokar